### PR TITLE
Resolves #2024

### LINKED ISSUE #2024: Get-vhd on a live vhdx
If the vhd/vhdx is in use, then can only run get-vhd on the system that has the vhd loaded or is running the VM that is using the vhd.
All other systems eg other nodes of a cluster with vhd's on shared storage, will give "The operation cannot be performed while the object is in use"

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7de2b1ce-c9e0-f7be-d8f1-9eb0ebd82677
* Version Independent ID: 776cd256-d681-539c-999e-e281be147647
* Content: [Get-VHD (hyper-v)](https://docs.microsoft.com/en-us/powershell/module/hyper-v/get-vhd?view=win10-ps)
* Content Source: [docset/windows/hyper-v/Get-VHD.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/hyper-v/Get-VHD.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Add note in description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2024

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/hyper-v/Get-VHD.md [hunk 1]
```diff
@@ -46,39 +46,42 @@ Get-VHD [-VMId] <Guid[]> [-CimSession <CimSession[]>] [-ComputerName <String[]>]
 ## DESCRIPTION
 The **Get-VHD** cmdlet gets the virtual hard disk object associated with a virtual hard disk.
 
+> [!Note]
+> When the VHD is in use (e.g by a VM or mounted in OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. All others will get and error that it is in use.
```
Comment (dariomws) [inline docset/windows/hyper-v/Get-VHD.md:50]: Same here, I think we cannot use standard note format because of Get-Help.

**Review by get-itips — COMMENTED** [review]
what is "all others" here?
agree with @dariomws

**Comment by velkovb** [comment]
@get-itips Every other server trying to run **Get-VHD** against the file. Really not sure who to say it appropriately.

**Comment by get-itips** [comment]
> @get-itips Every other server trying to run **Get-VHD** against the file. Really not sure who to say it appropriately.

Ok probably @JohanFreelancer9 will figure it out :)

File: docset/windows/hyper-v/Get-VHD.md [hunk 2]
```diff
@@ -46,39 +46,42 @@ Get-VHD [-VMId] <Guid[]> [-CimSession <CimSession[]>] [-ComputerName <String[]>]
 ## DESCRIPTION
 The **Get-VHD** cmdlet gets the virtual hard disk object associated with a virtual hard disk.
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Get-VHD.md:49]:
```suggestion
```

File: docset/windows/hyper-v/Get-VHD.md (hunk 1 above)
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Get-VHD.md:50]:
```suggestion
Note that when the VHD is in use (for example by a VM or mounted in the OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. Any other server trying to run the Get-VHD cmdlet will get an error that it is in use. 
```

File: docset/winserver2012r2-ps/hyper-v/Get-VHD.md [hunk 3]
```diff
@@ -42,39 +42,42 @@ Get-VHD [-VMId] <Guid[]> [-ComputerName <String[]>] [<CommonParameters>]
 ## DESCRIPTION
 The **Get-VHD** cmdlet gets the virtual hard disk object associated with a virtual hard disk.
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/hyper-v/Get-VHD.md:45]:
```suggestion
```

File: docset/winserver2012r2-ps/hyper-v/Get-VHD.md [hunk 4]
```diff
@@ -42,39 +42,42 @@ Get-VHD [-VMId] <Guid[]> [-ComputerName <String[]>] [<CommonParameters>]
...
+> When the VHD is in use (e.g by a VM or mounted in OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. All others will get and error that it is in use.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/hyper-v/Get-VHD.md:46]:
```suggestion
Note that when the VHD is in use (for example by a VM or mounted in the OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. Any other server trying to run the Get-VHD cmdlet will get an error that it is in use.  
```

File: docset/winserver2012-ps/hyper-v/Get-VHD.md [hunk 5]
```diff
@@ -14,57 +14,60 @@ Gets the virtual hard disk object associated with a virtual hard disk.
...
-### UNNAMED_PARAMETER_SET_3
+### VMId
 ```
 Get-VHD [-VMId] <Guid[]> [-ComputerName <String[]>]
 ```
 
 ## DESCRIPTION
 The **Get-VHD** cmdlet gets the virtual hard disk object associated with a virtual hard disk.
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Get-VHD.md:35]:
```suggestion
```

File: docset/winserver2012-ps/hyper-v/Get-VHD.md [hunk 6]
```diff
@@ -14,57 +14,60 @@ Gets the virtual hard disk object associated with a virtual hard disk.
...
+> When the VHD is in use (e.g by a VM or mounted in OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. All others will get and error that it is in use.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/hyper-v/Get-VHD.md:36]:
```suggestion
Note that when the VHD is in use (for example by a VM or mounted in the OS) and it is on shared storage, the **Get-VHD** cmdlet can only access it from the host that is using it. Any other server trying to run the Get-VHD cmdlet will get an error that it is in use. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**